### PR TITLE
Clean up: moving deprecated components to méli-mélo

### PR DESCRIPTION
**Pull request by Garneauma**
This is a clean up PR to deprecate some demoted components and move some already deprecated components to the experimentation section.

The following components/templates have been deprecated:
- Follow us variation, use [GC Follow us](https://wet-boew.github.io/GCWeb/components/gc-follow-us/gc-follow-us-en.html) variation instead
- Page details version 3.0 and 3.1, use [Page details version 3.2](https://wet-boew.github.io/GCWeb/sites/page-details/page-details-default-en.html) instead
- Theme and Topic template version 1, use [Topic template version 2.1](https://wet-boew.github.io/GCWeb/templates/topic/topic-en.html) instead
- Footer version 6.0, use [Footer version 7.0](https://wet-boew.github.io/GCWeb/sites/footers/no-footer-contextual-en.html) instead

The following components/theme have been moved to the experimentation section:
- Chat wizard
- Background image heading
- Light warning alert icon
- Pink day theme

Note: the Footer version 1 through 5 working examples have been removed from the codebase as they have been deprecated from more than a year.

### CODE REVIEW

File: components/gc-follow-us/_base.scss [hunk 1]
```diff
@@ -280,13 +279,6 @@ $social-media-icons-size: 38px;
 		@extend %social-media-icons-logo-instagram;
 	}
 
-	.googleplus { //This is deprecated and will be removed from later versions
-		@extend %social-media-icons-followus-logo-properties;
-		@extend %social-media-icons-logo-googleplus;
-		background-repeat: no-repeat;
-		background-size: 35px 35px;
-	}
```
Comment (duboisp) [inline components/gc-follow-us/_base.scss:288]: Can you duplicate the code about "googleplus" into the "deprecated" méli-mélo project. So for the one that want to use it, they will be able to use the méli-mélo compilation.

I know that was already mentioned in the code, but it was never officially communicated. Thanks

File: components/gc-follow-us/_base.scss [hunk 2]
```diff
@@ -2,6 +2,7 @@
  Social Media Icons, follow-us
 */
 
+// START: Deprecated as of version 16.0.0
```
Comment (duboisp) [inline components/gc-follow-us/_base.scss:5]: Deprecated code should be cloned into the "deprecated" méli-mélo project.

**Review by duboisp — CHANGES_REQUESTED** [review]
Partial and incomplete review, I only checked the follow-us deprecation

**Review by duboisp — CHANGES_REQUESTED** [review]
Je pensais que les fonctionnalités provisionnelles retiré seraient tous contenu dans "deprecated". La raison principal de leur déplacement c'est parce qu'il y a eu aucune action récente démontrant l'intérêt de les stabiliser officiellement

File: components/_provisional.scss
Comment (duboisp) [inline components/_provisional.scss:1]: Can you add a comment, start and end, to indicate the CSS block that has been deprecated?

File: méli-mélo/deprecated/style.css [hunk 3]
```diff
@@ -9,3 +9,31 @@
...
+	background-position: center center;
+	background-repeat: no-repeat;
+	background-size: 35px 35px;
+	display: inline;
+	min-height: 27px;
+	min-width: 27px;
+	position: relative;
+	vertical-align: text-bottom;
+}
```
Comment (duboisp) [inline méli-mélo/deprecated/style.css:39]: We should have a copy of all the depreceated css in gc-follow-us/_base.scss here

File: components/gc-follow-us/_base.scss [hunk 4]
```diff
@@ -44,10 +45,6 @@
...
-}
-
```
Comment (duboisp) [inline components/gc-follow-us/_base.scss:50]: Keep this here, but we should have a copy in the depreceated/style.css

File: méli-mélo/deprecated/assets/x-blue-line.png
Comment (duboisp) [inline méli-mélo/deprecated/assets/x-blue-line.png:1]: We can not move this asset, it need to remain in our dist until it's formal removal. But please keep a copy the file in the méli-mélo deprecated.

File: méli-mélo/deprecated/chatwizard/chatwizard-en.html
Comment (duboisp) [inline méli-mélo/deprecated/chatwizard/chatwizard-en.html:1]: You can move those working example page and only change the link that was in the index.json-ld files

Comment (duboisp) [inline méli-mélo/deprecated/chatwizard/chatwizard-en.html:1]: Also, the state in the index.json-ld for Chat wizard must be updated for "Deprecated"

**Review by duboisp — CHANGES_REQUESTED** [review]
Partial review.

We will need a note that those image file are deprecated and are going to removed from our distribution.
* x-blue-line.png

File: méli-mélo/deprecated/deprecated.html [hunk 5]
```diff
@@ -41,3 +41,71 @@ <h3>Code</h3>
...
+		<li><a href="#" class="email" rel="external"><span class="wb-inv">Email subscription</span></a></li>
+		<li><a href="#" class="periscope" rel="external"><span class="wb-inv">Periscope</span></a></li>
+		<li><a href="#" class="googleplus" rel="external"><span class="wb-inv">Google+</span></a></li>
+	</ul>
+</section>
+
+<h2>Background image heading <span class="label label-warning">Deprecated as of v16.0.0</span></h2>
+<p>Add a background image to a heading.</p>
+
+<h3>Working example</h3>
```
Comment (duboisp) [inline méli-mélo/deprecated/deprecated.html:66]: Remove the existing working example that we have in our current provisional page. And in the provisional page link to this deprecated page.

File: méli-mélo/deprecated/pinkday/docs-en.html
Comment (duboisp) [inline méli-mélo/deprecated/pinkday/docs-en.html:1]: Update the link in the index.json-ld of template/thematic/pink-day.html

And delete the HTML page only that is in the template/thematic/ folder

File: méli-mélo/deprecated/pinkday/meta.md
Comment (duboisp) [inline méli-mélo/deprecated/pinkday/meta.md:1]: This file can be removed as pink day are not going to be an independent experimental project

**Review by duboisp — CHANGES_REQUESTED** [review]
Partial review

File: sites/page-details/index.json-ld [hunk 6]
```diff
@@ -110,12 +110,12 @@
 					"fr": { "href": "page-details-pft-fr.html", "text": "Détails de la page avec Outil de rétroaction sur la page" }
 				},
 				{
-					"en": { "href": "demoted/page-details-v3-en.html", "text": "Demoted - Page details version 3.0" },
-					"fr": { "href": "demoted/page-details-v3-fr.html", "text": "Rétrogradé - Détails de la page version 3.0" }
+					"en": { "href": "deprecated/page-details-v3-en.html", "text": "Deprecated - Page details version 3.0" },
+					"fr": { "href": "deprecated/page-details-v3-fr.html", "text": "Obsolète - Détails de la page version 3.0" }
```
Comment (duboisp) [inline sites/page-details/index.json-ld:114]: Let's add the GCWeb version of when we deprecated that pattern. It's to ease future clean up 

```suggestion
					"en": { "href": "deprecated/page-details-v3-en.html", "text": "Deprecated as GCWeb v16.0.0 - Page details version 3.0" },
					"fr": { "href": "deprecated/page-details-v3-fr.html", "text": "Obsolète à partir de GCWeb v16.0.0 - Détails de la page version 3.0" }
```

File: sites/page-details/index.json-ld [hunk 7]
```diff
@@ -110,12 +110,12 @@
...
 				},
 				{
-					"en": { "href": "demoted/page-details-v3.1-en.html", "text": "Demoted - Page details version 3.1" },
-					"fr": { "href": "demoted/page-details-v3.1-fr.html", "text": "Rétrogradé - Détails de la page version 3.1" }
+					"en": { "href": "deprecated/page-details-v3.1-en.html", "text": "Deprecated - Page details version 3.1" },
+					"fr": { "href": "deprecated/page-details-v3.1-fr.html", "text": "Obsolète - Détails de la page version 3.1" }
```
Comment (duboisp) [inline sites/page-details/index.json-ld:118]:
```suggestion
					"en": { "href": "deprecated/page-details-v3.1-en.html", "text": "Deprecated as GCWeb v16.0.0 - Page details version 3.1" },
					"fr": { "href": "deprecated/page-details-v3.1-fr.html", "text": "Obsolète à partir de GCWeb v16.0.0 - Détails de la page version 3.1" }
```

File: sites/page-details/deprecated/page-details-v3-fr.html
Comment (duboisp) [inline sites/page-details/deprecated/page-details-v3-fr.html:1]: Change the word "retrogradé" in the page title for "obsolète"

File: sites/page-details/deprecated/page-details-v3.1-en.html
Comment (duboisp) [inline sites/page-details/deprecated/page-details-v3.1-en.html:1]: Change the word "demoted" in the page title for "deprecated"

File: sites/page-details/deprecated/page-details-v3.1-fr.html
Comment (duboisp) [inline sites/page-details/deprecated/page-details-v3.1-fr.html:1]: Change the word "retrogradé" in the page title for "obsolète"

File: templates/institutional-landing/reports/a11y-1.json [hunk 8]
```diff
@@ -6,7 +6,7 @@
   ],
   "earl:subject": {
     "@id": "_:subject",
-    "dct:references": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/demoted/institution-landing-en.html",
+    "dct:references": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/deprecated/institution-landing-en.html",
```
Comment (duboisp) [inline templates/institutional-landing/reports/a11y-1.json:9]: Don't update the link in the accessibility report. They are not designed to be maintainable content but more to reflect the state at that specific moment of time of when the report was created.

```suggestion
    "dct:references": "https://wet-boew.github.io/GCWeb/templates/institutional-landing/demoted/institution-landing-en.html",
```

File: templates/topic/index.json-ld [hunk 9]
```diff
@@ -299,13 +299,13 @@
 			"example": {
 				"en": [
 					{
-						"href": "demoted/theme-topic-en.html",
+						"href": "deprecated/theme-topic-en.html",
 						"text": "Theme - Topic page version 2 (beta)"
```
Comment (duboisp) [inline templates/topic/index.json-ld:303]: We are missing a change set to hold the state of those iteration.

For now, let update the title and clearly indicate they are deprecated.

```suggestion
						"href": "deprecated/theme-topic-en.html",
						"text": "Deprecated as GCWeb v16.0.0 - Theme - Topic page version 2 (beta)"
```

File: templates/topic/index.json-ld [hunk 10]
```diff
@@ -41,22 +41,22 @@
 				"path": "topic-stackedheader-fr.html"
 			},
 			{
-				"title": "Theme/Topic page - Demoted",
+				"title": "Theme/Topic page - Deprecated",
```
Comment (duboisp) [inline templates/topic/index.json-ld:44]:
```suggestion
				"title": "Theme/Topic page - Deprecated as GCWeb v16",
```

File: templates/topic/index.json-ld [hunk 11]
```diff
@@ -41,22 +41,22 @@
...
 				"language": "en",
-				"path": "demoted/theme-topic-en.html"
+				"path": "deprecated/theme-topic-en.html"
 			},
 			{
-				"title": "Page de sujet/thème - Rétrogradée",
+				"title": "Page de sujet/thème - Obsolète",
```
Comment (duboisp) [inline templates/topic/index.json-ld:49]:
```suggestion
				"title": "Page de sujet/thème - Obsolète à partir de GCWeb v16",
```

File: templates/topic/index.json-ld [hunk 12]
```diff
@@ -41,22 +41,22 @@
...
+				"path": "deprecated/theme-topic-fr.html"
 			},
 			{
 				"title": "Lowest topic (with secondary navigation) - Deprecated",
 				"language": "en",
 				"path": "../lowest-topic/index-en.html"
 			},
 			{
-				"title": "Sujet de plus bas niveau (avec une navigation secondaire) - Dépréciée",
+				"title": "Sujet de plus bas niveau (avec une navigation secondaire) - Obsolète",
```
Comment (duboisp) [inline templates/topic/index.json-ld:59]:
```suggestion
				"title": "Sujet de plus bas niveau (avec une navigation secondaire) - Obsolète à partir de GCWeb v16",
```

File: templates/topic/index.json-ld [hunk 13]
```diff
@@ -41,22 +41,22 @@
...
 			},
 			{
-				"title": "Page de sujet/thème - Rétrogradée",
+				"title": "Page de sujet/thème - Obsolète",
 				"language": "fr",
-				"path": "demoted/theme-topic-fr.html"
+				"path": "deprecated/theme-topic-fr.html"
 			},
 			{
 				"title": "Lowest topic (with secondary navigation) - Deprecated",
```
Comment (duboisp) [inline templates/topic/index.json-ld:54]:
```suggestion
				"title": "Lowest topic (with secondary navigation) - Deprecated as GCWeb v16",
```

File: templates/topic/index.json-ld [hunk 14]
```diff
@@ -319,7 +319,7 @@
 			"example": {
 				"en": [
 					{
-						"href": "demoted/topic-en.html",
+						"href": "deprecated/topic-en.html",
 						"text": "Topic page version 1"
```
Comment (duboisp) [inline templates/topic/index.json-ld:323]:
```suggestion
						"href": "deprecated/topic-en.html",
						"text": "Deprecated as of GCWeb v16 - Topic page version 1"
```

File: templates/thematic/index.json-ld [hunk 15]
```diff
@@ -18,26 +18,38 @@
 	"componentName": "thematic",
 	"status": "provisional",
 	"pages": {
+		"docs": [
+			{
+				"title": "Pink Day",
```
Comment (duboisp) [inline templates/thematic/index.json-ld:23]:
```suggestion
				"title": "Pink Day - Deprecated as of GCWeb v16",
```

File: templates/thematic/index.json-ld [hunk 16]
```diff
@@ -18,26 +18,38 @@
...
+			},
+			{
+				"title": "Journée rose",
+				"language": "fr",
+				"path": "../../méli-mélo/deprecated/pinkday/docs-fr.html"
+			}
+		],
 		"examples": [
 			{
 				"title": "Pink Day",
```
Comment (duboisp) [inline templates/thematic/index.json-ld:35]:
```suggestion
				"title": "Pink Day - Deprecated as of GCWeb v16",
```

File: templates/thematic/index.json-ld [hunk 17]
```diff
@@ -18,26 +18,38 @@
...
+				"path": "../../méli-mélo/deprecated/pinkday/pink-day-en.html"
 			},
 			{
 				"title": "Journée rose",
 				"language": "fr",
-				"path": "pink-day-fr.html"
+				"path": "../../méli-mélo/deprecated/pinkday/pink-day-fr.html"
 			},
 			{
 				"title": "Pink Day - Home",
```
Comment (duboisp) [inline templates/thematic/index.json-ld:45]:
```suggestion
				"title": "Pink Day - Home - Deprecated as of GCWeb v16",
```

File: templates/thematic/index.json-ld [hunk 18]
```diff
@@ -18,26 +18,38 @@
...
 				"language": "en",
-				"path": "pink-day-home-en.html"
+				"path": "../../méli-mélo/deprecated/pinkday/pink-day-home-en.html"
 			},
 			{
 				"title": "Journée rose - Accueil",
```
Comment (duboisp) [inline templates/thematic/index.json-ld:50]:
```suggestion
				"title": "Journée rose - Accueil - Obsolète à partir de GCWeb v16",
```

File: templates/thematic/index.json-ld [hunk 19]
```diff
@@ -18,26 +18,38 @@
...
+		],
 		"examples": [
 			{
 				"title": "Pink Day",
 				"language": "en",
-				"path": "pink-day-en.html"
+				"path": "../../méli-mélo/deprecated/pinkday/pink-day-en.html"
 			},
 			{
 				"title": "Journée rose",
```
Comment (duboisp) [inline templates/thematic/index.json-ld:40]:
```suggestion
				"title": "Journée rose - Obsolète à partir de GCWeb v16",
```

File: templates/thematic/index.json-ld [hunk 20]
```diff
@@ -18,26 +18,38 @@
...
 	"status": "provisional",
 	"pages": {
+		"docs": [
+			{
+				"title": "Pink Day",
+				"language": "en",
+				"path": "../../méli-mélo/deprecated/pinkday/docs-en.html"
+			},
+			{
+				"title": "Journée rose",
```
Comment (duboisp) [inline templates/thematic/index.json-ld:28]:
```suggestion
				"title": "Journée rose - Obsolète à partir de GCWeb v16",
```

**Review by duboisp — CHANGES_REQUESTED** [review]
A few little content adjustment and some change for the CSS copied over the deprecated css file.

File: components/gc-follow-us/_base.scss [hunk 21]
```diff
@@ -280,13 +283,6 @@ $social-media-icons-size: 38px;
 		@extend %social-media-icons-logo-instagram;
 	}
 
-	.googleplus { //This is deprecated and will be removed from later versions
-		@extend %social-media-icons-followus-logo-properties;
-		@extend %social-media-icons-logo-googleplus;
-		background-repeat: no-repeat;
-		background-size: 35px 35px;
-	}
-
```
[the remark this answers is not in the document]
Comment (Garneauma) [reply components/gc-follow-us/_base.scss:289]: I thought it was already deprecated, so we were at the step of removing it.

**Review by duboisp — APPROVED** [review]
Reviewed and tested locally. Everything look good.

@ouafaaetta note this PR do impact a lot of component/feature and provisional feature. Take the time to get the full list.

This is major change as it mark a few component as "Deprecated" and it remove the support for the old site footer which was deprecated a year and half ago.

**Pull request merged**